WireAuthentication/Sources/WireAuthenticationUI/Views/DetermineAuthMethod/DetermineAuthMethodView.swift

```diff
@@ -70,6 +70,8 @@ package struct DetermineAuthMethodView: View {
                         title: L10n.Authentication.Identity.Input.Field.title,
                         string: $viewModel.emailOrSSOCode
                     )
+                    .autocapitalization(.none)
+                    .keyboardType(.emailAddress)
```

Comment:
`question:` But this input field is not only for email, it's for SSO code as well. Are you sure we should specify the keyboardType?